### ISSUE TITLE
[TF 2.0 API Docs] tf.keras.activations.softmax

### ISSUE THREAD

**Issue by BlackHoundNate** [tensorflow/tensorflow#26372]
<em>Please make sure that this is a documentation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>

**System information**
- TensorFlow version: 2.0
- Doc Link: https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/activations/softmax

**Describe the documentation issue**

* The softmax activation function is not described in detail, and there is no recommendation about when to use it.

* There is no usage example.

* The description of the returned value could be more useful.

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**

Yes.

**Comment by Jaskaran170599** [comment]
I would like to contribute on this.

**Comment by jvishnuvardhan** [comment]
@Jaskaran170599 Please feel free to contribute through PRs. Thanks!

**Comment by Jaskaran170599** [comment]
Looks like the PR is merged already , please close the issue.

**Comment by jvishnuvardhan** [comment]
I am closing this as this was already updated in the TF website [here](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/activations/softmax).  

Please feel free to reopen if I am mistaken. Thanks!
